airbyte-integrations/connectors/source-freshcaller/Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /airbyte/integration_code
 # upgrade pip to the latest version
 RUN apk --no-cache upgrade \
     && pip install --upgrade pip \
-    && apk --no-cache add tzdata build-base
+    && apk --no-cache add tzdata build-base libffi-dev
```

Comment:
Why do we need `libffi-dev` here?

Reply:
We shouldn't need this after migrating this connector to the standard bae image as you mentioned. I'll keep my eyes peeled for when that gets merged.